ember-addon-main.js

```diff
@@ -1,6 +1,23 @@
 'use strict';
 
 let VersionChecker = require('ember-cli-version-checker');
+let babelPluginPath = require.resolve('./src/babel-plugin.js');
+
+function isAddTOCNamesForInspectorBabelPluginRegistered(plugins, root) {
```

Comment:
why is this function called this? what's a TOCNames for Inspector?

Reply:
TOC is for Template Only Component, however we can name it whatever it makes sense, maybe we could do just `isBabelPluginRegistered` for simplicity as it's the only plugin (if I'm not mistaken)? WDYT @NullVoxPopuli ?

Reply:
ya I like that gives just the information we need